src/MaterialDesignThemes.Wpf/Converters/SliderValueLabelPositionConverter.cs

```diff
@@ -4,7 +4,7 @@
 namespace MaterialDesignThemes.Wpf.Converters;
 
 [ValueConversion(typeof(double), typeof(double), ParameterType = typeof(Orientation))]
-internal class SliderValueLabelPositionConverter : IValueConverter
+public class SliderValueLabelPositionConverter : IValueConverter
```

Comment:
I propose moving this inside of a new `MaterialDesignThemes.Wpf.Converters.Internal` namespace.